debug_scripts/send_validator_logs.py

```diff
@@ -0,0 +1,48 @@
+#!/bin/python3
+import argparse
+import boto3
+import datetime
+import io
+import urllib.parse
+
+parser = argparse.ArgumentParser(description='Send logs to near.')
+parser.add_argument('--log_file', type=str, help='Absolute path to log file.', required=True)
+parser.add_argument('--account', type=str, help='Near account id.', required=True)
+parser.add_argument('--time_range', type=int, help='Filter logs for last x seconds.', required=True)
```

Comment:
nit: this is not a 'range' - as we're passing a single value.

you can call it 'recent_seconds' or 'last_seconds'

Reply:
updated